CHANGELOG.md

```diff
@@ -1,60 +1,3 @@
-# Changelog
```

Comment:
Why is Changelog.md content getting removed

Reply:
Resolved in ed08b2785315fd78936fab8a793058cc364bc353